Traceback (most recent call last):
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/obm/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
vector = [1,1]
qc.initialize(vector, 0)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mQiskitError[0m                               Traceback (most recent call last)
Input [0;32mIn [13][0m, in [0;36m<cell line: 2>[0;34m()[0m
[1;32m      1[0m vector [38;5;241m=[39m [[38;5;241m1[39m,[38;5;241m1[39m]
[0;32m----> 2[0m [43mqc[49m[38;5;241;43m.[39;49m[43minitialize[49m[43m([49m[43mvector[49m[43m,[49m[43m [49m[38;5;241;43m0[39;49m[43m)[49m

File [0;32m~/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/qiskit/extensions/quantum_initializer/initializer.py:191[0m, in [0;36minitialize[0;34m(self, params, qubits)[0m
[1;32m    188[0m     qubits [38;5;241m=[39m [qubits]
[1;32m    189[0m num_qubits [38;5;241m=[39m [38;5;28mlen[39m(qubits) [38;5;28;01mif[39;00m [38;5;28misinstance[39m(params, [38;5;28mint[39m) [38;5;28;01melse[39;00m [38;5;28;01mNone[39;00m
[0;32m--> 191[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39mappend([43mInitialize[49m[43m([49m[43mparams[49m[43m,[49m[43m [49m[43mnum_qubits[49m[43m)[49m, qubits)

File [0;32m~/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/qiskit/extensions/quantum_initializer/initializer.py:57[0m, in [0;36mInitialize.__init__[0;34m(self, params, num_qubits)[0m
[1;32m     36[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, params, num_qubits[38;5;241m=[39m[38;5;28;01mNone[39;00m):
[1;32m     37[0m     [38;5;124mr[39m[38;5;124;03m"""Create new initialize composite.[39;00m
[1;32m     38[0m 
[1;32m     39[0m [38;5;124;03m    Args:[39;00m
[0;32m   (...)[0m
[1;32m     55[0m [38;5;124;03m            and the remaining 3 qubits to be initialized to :math:`|0\rangle`.[39;00m
[1;32m     56[0m [38;5;124;03m    """[39;00m
[0;32m---> 57[0m     [38;5;28mself[39m[38;5;241m.[39m_stateprep [38;5;241m=[39m [43mStatePreparation[49m[43m([49m[43mparams[49m[43m,[49m[43m [49m[43mnum_qubits[49m[43m)[49m
[1;32m     59[0m     [38;5;28msuper[39m()[38;5;241m.[39m[38;5;21m__init__[39m([38;5;124m"[39m[38;5;124minitialize[39m[38;5;124m"[39m, [38;5;28mself[39m[38;5;241m.[39m_stateprep[38;5;241m.[39mnum_qubits, [38;5;241m0[39m, [38;5;28mself[39m[38;5;241m.[39m_stateprep[38;5;241m.[39mparams)

File [0;32m~/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/qiskit/circuit/library/data_preparation/state_preparation.py:99[0m, in [0;36mStatePreparation.__init__[0;34m(self, params, num_qubits, inverse, label)[0m
[1;32m     96[0m [38;5;28mself[39m[38;5;241m.[39m_from_label [38;5;241m=[39m [38;5;28misinstance[39m(params, [38;5;28mstr[39m)
[1;32m     97[0m [38;5;28mself[39m[38;5;241m.[39m_from_int [38;5;241m=[39m [38;5;28misinstance[39m(params, [38;5;28mint[39m)
[0;32m---> 99[0m num_qubits [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43m_get_num_qubits[49m[43m([49m[43mnum_qubits[49m[43m,[49m[43m [49m[43mparams[49m[43m)[49m
[1;32m    101[0m params [38;5;241m=[39m [params] [38;5;28;01mif[39;00m [38;5;28misinstance[39m(params, [38;5;28mint[39m) [38;5;28;01melse[39;00m params
[1;32m    103[0m [38;5;28msuper[39m()[38;5;241m.[39m[38;5;21m__init__[39m([38;5;28mself[39m[38;5;241m.[39m_name, num_qubits, params, label[38;5;241m=[39m[38;5;28mself[39m[38;5;241m.[39m_label)

File [0;32m~/Prog/miniconda3/envs/qml/lib/python3.8/site-packages/qiskit/circuit/library/data_preparation/state_preparation.py:202[0m, in [0;36mStatePreparation._get_num_qubits[0;34m(self, num_qubits, params)[0m
[1;32m    200[0m     [38;5;66;03m# Check if probabilities (amplitudes squared) sum to 1[39;00m
[1;32m    201[0m     [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m math[38;5;241m.[39misclose([38;5;28msum[39m(np[38;5;241m.[39mabsolute(params) [38;5;241m*[39m[38;5;241m*[39m [38;5;241m2[39m), [38;5;241m1.0[39m, abs_tol[38;5;241m=[39m_EPS):
[0;32m--> 202[0m         [38;5;28;01mraise[39;00m QiskitError([38;5;124m"[39m[38;5;124mSum of amplitudes-squared does not equal one.[39m[38;5;124m"[39m)
[1;32m    204[0m     num_qubits [38;5;241m=[39m [38;5;28mint[39m(num_qubits)
[1;32m    205[0m [38;5;28;01mreturn[39;00m num_qubits

[0;31mQiskitError[0m: 'Sum of amplitudes-squared does not equal one.'
QiskitError: 'Sum of amplitudes-squared does not equal one.'

